### PR TITLE
Theme: Revert normalize.css' link outline styles

### PR DESCRIPTION
**Pull request by EricDunsworth**
Imports WET's normalize.css overrides to obtain the fix.

Depends on wet-boew/wet-boew#9737.

### CODE REVIEW

**Comment by EricDunsworth** [comment]
This is failing to build atm since the file it's importing doesn't exist in GCWeb's WET dependency... yet.

Once wet-boew/wet-boew#9737 gets merged, GCWeb's WET dependency will need to be updated and this PR will need to be rebased to make it build-able.

Btw I haven't tried testing the fix in GCWeb first-hand. But the problem and fix are so obvious that I didn't feel like it was really necessary. I'd fully expect GCWeb to behave identically to WET for this.

**Comment by Garneauma** [comment]
Pre-approved upon merge of PR #9737 and successful review.

**Review by Garneauma — APPROVED** [review]
I manually updated WET-BOEW's reference in package.json and package-lock.json and applied this change. It works as expected.

**Comment by EricDunsworth** [comment]
Actually - I'll hold off until GCWeb's WET dependency has been updated (doesn't look like it's happened yet).

**Comment by EricDunsworth** [comment]
Rebased now :).

**Pull request merged**
